integration/default_set.yaml

```diff
@@ -41,7 +41,7 @@ tests:
       - command: export -N ascii 'orig_bytes > 1k && orig_bytes < 1Ki'
       - command: export -N ascii ':string == "OrfTtuI5G4e" || :port == 67/udp'
       - command: export -N ascii '#type == "zeek.conn" && resp_h == 192.168.1.104'
-      - command: export -N ascii '#type != "zeek.conn"'
+      - command: export -N ascii '#type != "zeek.conn" && \#type != "vast.account"'
```

Comment:
I find this more idiomatic: `#type !in {"zeek.conn", "vast.account"}`.

Reply:
This does currently not work.